src/app/profile/_components/FilledProfile/FiledProfile.tsx

```diff
@@ -0,0 +1,184 @@
+'use client'
+import { useState, useEffect } from 'react'
+import {
+  authenticateUser,
+  getUserData,
+  UserData,
+} from '@/services/authAndUserService'
+import Image from 'next/image'
+import Container from '@/components/Container/Container'
+
+const FiledProfile = () => {
+  const [userData, setUserData] = useState<UserData | null>(null)
+  const userId = '11111111-1111-1111-1111-111111111111'
+
+  console.log(userData)
```

Comment:
I will temporarily leave the console in the code because corses are not working and I can't use custom data! if you remove the console linter warns about an unused variable! Once the server is working on the client, I will remove it immediately.